v3/src/components/graph/components/scatterdots.tsx

```diff
@@ -171,7 +171,7 @@ export const ScatterDots = memo(function ScatterDots(props: IProps) {
   }, [refreshPointPositionsD3, refreshPointPositionsSVG])
 
   usePlotResponders({
-    dataset, xAxisModel, yAxisModel, primaryAttrID, secondaryAttrID, layout,
+    graphModel, primaryAttrID, secondaryAttrID, layout, dotsRef,
```

Comment:
Remove `xAxisModel` and `yAxisModel` from props above now that they're no longer used.